code/game/objects/structures/vehicles/snowmobile.dm

```diff
@@ -24,24 +66,24 @@
 //Universal key snowmobiles
 /obj/item/key/snowmobile/universal
 	name = "universal snowmobile key"
-	desc = "Someone has to do it. This key has a universal bitting."
+	desc = "An universal ignition key for use with Nanotrasen's snowmobile brand, NT patented cheap ignition lock systems means that any snowmobile can be driven with it. Useful to have when everything goes to hell."
 
 /obj/item/key/snowmobile/universal/initialize()
 	return
 
 /obj/structure/bed/chair/vehicle/snowmobile/universal
-	desc = "There's something out there, and now you can catch it. This snowmobile uses a universal key."
+	desc = "A Nanotrasen motorized vehicle with kelvar tracks designed for use in snowy environments. It's also equipped to handle frozen terrain and road surfaces, but will perform poorly indoors. This one has NT patented universal ignition for easier vehicle 'borrowing'."
 	keytype = /obj/item/key/snowmobile/universal
 
 /obj/structure/bed/chair/vehicle/snowmobile/universal/set_keys()
 	if(keytype && !vin)
-		new keytype(loc)
+		heldkey = new keytype(src)
 
 /obj/structure/bed/chair/vehicle/snowmobile/getMovementDelay()
 	var/turf/T = loc
-	if(is_type_in_list(T,approved_terrain))
+	if(is_type_in_list(T, approved_terrain))
 		return 1
 	else
-		var/list/dragsounds = list('sound/misc/metal_drag1.ogg','sound/misc/metal_drag2.ogg','sound/misc/metal_drag3.ogg')
-		playsound(src, pick(dragsounds), 50, 1)
-		return 5 //It's not designed to move this way!
\ No newline at end of file
+		var/list/dragsounds = list('sound/misc/metal_drag1.ogg', 'sound/misc/metal_drag2.ogg', 'sound/misc/metal_drag3.ogg')
+		playsound(src, pick(dragsounds), 10, 1) //The scratching sound is VERY loud, obnoxious and repeats often, so make sure the volume is low
+		return 7 //It's not designed to move this way!
```

Comment:
undocumented speed nerf

Reply:
Oh shit that's the speed, let me fix that